["```py\n3 + 4\n```", "```py\n7\n```", "```py\nx = 3\ny = 14\n```", "```py\ny = y – current_temperature\n```", "```py\nresult = x / y\n```", "```py\nmy_byte = 127\n```", "```py\nmy_byte = my_byte + 1\n```", "```py\nx = 1.3\ny = 1.1\nprint x + y\n```", "```py\n2.4\n```", "```py\n2.4000000000000004\n```", "```py\nif age < 18 then\n   …\nend_if\n```", "```py\nif age > 18 then\n   …\nend_if\n```", "```py\nif age >= 18 then\n   …\nend_if\n```", "```py\nif age = 20 then\n   …\nend_if\n```", "```py\nif age > 12 or age < 20 then\n   …\nend_if\n```", "```py\nif age > 12 and age < 20 then\n   …\nend_if\n```", "```py\nprint \"Hello\n```", "```py\nSyntaxError: EOL while scanning string literal in line 1 column 12\n1:12 syntax error: unexpected newline\nCompilation error (line 1, col 12): Newline in constant\nerror: unknown: Unterminated string constant (1:12)\n```", "```py\n1apple = 1\n```", "```py\nSyntaxError: invalid syntax in line 1 column 2\n1:2 syntax error: unexpected apple at end of statement\nCompilation error (line 1, col 2): Identifier expected\nerror: unknown: Identifier directly after number (1:2)\n```", "```py\nif age > 12 or age < 20 then\n   …\nend_if\n```", "```py\nYou are a teenager.\n```", "```py\nYou are a teenager.\n```", "```py\nYou are a teenager.\n```", "```py\nif age > 12 or age < 20:\n```", "```py\nnames = [\"Anna\", \"Bob\", \"Cara\", \"David\"]\n```", "```py\nprint names[2]\n```", "```py\nCara\n```", "```py\nprint names[6]\n```", "```py\nIndexOutOfRangeException: Index was outside the bounds of the array at line 2.\n```", "```py\nfunction c()\n    result = 10 / 0\nend_function\nfunction b()\n    c()\nend_function\nfunction a()\n    b()\nend_function\nfunction main()\n    a()\nend_function\n```", "```py\nCallstack\nfunction c() at line 2\nfunction b() at line 6\nfunction a() at line 10\nfunction main() at line 14\nZeroDivisionError: division by zero\n```", "```py\nfunction c(x, y)\n    result = x / y\nend_function\nfunction b()\n    c(10, 0)\nend_function\nfunction a()\n    b()\nend_function\nfunction main()\n    a()\nend_function\n```", "```py\nfunction c(x, y)\n    result = x / y\nend_function\n```", "```py\nfunction c(x, y)\n    // The function does some things ere\n    result = x / y\n    // And even more things here\n    // It might even return a value\nend_function\n```", "```py\nfunction calculate(x, y)\n    if y == 0 then\n\t  // y is zero, so we cannot perform the division\n    end_if\n    // The function does some things ere\n    result = x / y\n    // And even more things here\n    // It might even return a value\nend_function\n```", "```py\ntry\n    calculate(10, 0)\ncatch ZeroDivisionError\n   // We will end up here if we get a ZeroDivisionError    // exception\nend_catch\n```", "```py\nfunction calculate(x, y)\n    if y == 0 then\n\t  throw ValueError(\"Second argument cannot be zero\")\n    end_if\n    // The function does some things ere\n    result = x / y\n    // And even more things here\n    // It might even return a value\nend_function\n```", "```py\ntry\n    calculate(10, 0)\ncatch ValueError\n   // We will end up here if we get a ValueError exception\nend_catch\n```"]